$NetBSD: patch-Makefile,v 1.1 2020/12/13 15:47:18 nia Exp $

1. Honour CFLAGS and LDFLAGS
2. Don't compress the output binary (upx)

--- Makefile.orig	2020-07-27 10:03:51.000000000 +0000
+++ Makefile
@@ -50,9 +50,9 @@ WINMINGWPREFIX ?= i686-w64-mingw32
 # configuration end
 ###############################################################################
 
-CFLAGS = -Wall -ansi -pedantic -Wno-long-long
+CFLAGS += -Wall -ansi -pedantic -Wno-long-long
 
-LDFLAGS =
+LDFLAGS +=
 
 # iin_hdloader.c iin_net.c
 SOURCES = hdl_dump.c \
@@ -184,9 +184,6 @@ rsrc.o: rsrc.rc
 $(BINARY): $(OBJECTS)
 	@echo -e "\tLNK $@"
 	@$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS)
-ifeq ($(RELEASE), yes)
-	-@upx -q -9 $@ > /dev/null
-endif
 
 
 %.o : %.c
